core_lib/structure/layer.cpp

```diff
@@ -474,7 +474,7 @@ void Layer::mouseRelease( QMouseEvent* event, int frameNumber )
 
 void Layer::editProperties()
 {
-    QRegExp regex("([\uFFEF-\uFFFF])+");
+    QRegExp regex("([\\xFFEF-\\xFFFF])+");
```

Comment:
The documentation doesn’t mention braces and it seemed to work fine when testing. What you are probably thinking of is QRegularExpression which uses PCRE, see http://www.pcre.org/current/doc/html/pcre2syntax.html#SEC3. QRegExp uses its own syntax.